### PR TITLE
[ICONS] fixed icon alignment inside a button

### PR DESCRIPTION
**Pull request by Shijir**
Tested on:

- Chrome
- Firefox
- Safari
- Edge
- IE11
- IE10

Signed-off-by: Shijir Tsogoo <stsogoo@vmware.com>

### CODE REVIEW

**Comment by adityarb88** [comment]
@Shijir: Please add more details on your PR description. Details like screenshots, which browsers you have tested this change on etc.

File: src/clarity/buttons/_buttons.clarity.scss [hunk 1]
```diff
@@ -265,13 +264,23 @@ $clr-button-appearance-map: (
         }
 
         &.btn-sm {
-            @include clr-getStylesFromMap($clr-button-appearance-map,standard);
+            @include clr-getStylesFromMap($clr-button-appearance-map, standard);
+
+            clr-icon {
```
Comment (adityarb88) [inline src/clarity/buttons/_buttons.clarity.scss]: Whats the size of the icons in regular buttons? Am assuming its the default size: 16px?

File: src/clarity/buttons/_buttons.clarity.scss [hunk 2]
```diff
@@ -265,13 +264,23 @@ $clr-button-appearance-map: (
...
         }
 
         &.btn-block {
             display: block;
             width: 100%;
             max-width: 100%;
         }
+
+        clr-icon {
+            vertical-align: text-bottom;
```
Comment (adityarb88) [inline src/clarity/buttons/_buttons.clarity.scss]: Does this alignment work on all browsers? I have noticed that the `vertical-align` & `margin` combo works a bit differently in IE and Firefox. Please confirm? If possible please paste screenshots.

File: src/clarity/buttons/_buttons.clarity.scss (hunk 1 above)
Comment (Shijir) [inline src/clarity/buttons/_buttons.clarity.scss]: Yes. The default is 16px.

**Comment by Shijir** [comment]
screenshot 2016-12-14 12 46 06

**Pull request merged**